extensions/PhabBugz/lib/Util.pm

```diff
@@ -423,11 +423,11 @@ sub get_attachment_revisions {
         }
 
         if (@revision_ids) {
-            @revisions = get_revisions_by_ids( \@revision_ids );
+            $revisions = get_revisions_by_ids( \@revision_ids );
         }
     }
 
-    return @revisions;
+    return @$revisions;
```

Comment:
I haven't run this -- did the return value of get_revisions_by_ids() change in this patch? or was this wrong before?

Reply:
This was wrong before. get_revisions_by_ids returns reference to a list but code expects get_attachment_revisions to return a list and not a ref so I had to fix this. Luckily the inconsistent code will be going away.